.github/workflows/pandora-ci.yml

```diff
@@ -25,28 +25,25 @@ concurrency:
 
 jobs:
   unit-tests:
-    permissions: write-all
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
Check your repository settings to prevent `Error: HttpError: Resource not accessible by integration` on step uploading the reports
![image](https://github.com/bitbrain/pandora/assets/347037/c265f9ea-8d2c-4068-a59f-3b4ce7503e77)